src/services/putData.js

```diff
@@ -0,0 +1,13 @@
+const postData = async (url, data) => {
```

Comment:
postData in putData file?  Maybe just create client.js with pust post and other methods inside?

Reply:
Yeah, I spotted this later and changed to putData :D